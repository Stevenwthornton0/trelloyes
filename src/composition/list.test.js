import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './list.js';


describe('list component', () => {
  const Card = [
    { title: 'First card', content: 'lorem ipsum' },
    { title: 'Second card', content: 'lorem ipsum' }
  ]

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List header='First list' cards={Card} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer
      .create(<List header='First list' cards={Card} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})