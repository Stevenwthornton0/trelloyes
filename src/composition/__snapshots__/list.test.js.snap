// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list component renders UI as expected 1`] = `
<section
  className="List"
>
  <header
    className="list-header"
  >
    <h2>
      First list
    </h2>
  </header>
  <div
    className="list-cards"
  >
    <div
      className="Card"
    >
      <h3
        className="title"
      >
        First card
      </h3>
      <p
        className="content"
      >
        lorem ipsum
      </p>
    </div>
    <div
      className="Card"
    >
      <h3
        className="title"
      >
        Second card
      </h3>
      <p
        className="content"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</section>
`;
